NEWS.md

```diff
@@ -7,13 +7,12 @@ Thanks to @seabbs, and @sbfnk and for [SACEMA](https://sacema.org) for hosting @
 ## Breaking changes
 
 * To enable enhance functionality `trunc_opts()` now takes a single argument (`dist`) which defines the truncation delay rather than a arbitary list of arguments (which was previously used to define the distribution.
-
+* Updated the handling of generation times in the renewal equation to be left truncation adjusted for the zeroth day. This more accurately the typical approach taken to estimate generation times but may slightly impact estimates vs those produced using previous versions.
```

Comment:
As mentioned in [this comment](https://github.com/epiforecasts/EpiNow2/pull/343#discussion_r1084398737) I think it might be more accurate to say it's indexed such that the first bin of the discretised distribution reflects the range [1,2] of the continuous version (but then also extends to [n, n + 1] as the last bin).

Reply:
so that is true but there is also an adjustment for left truncation in there as well no?

Reply:
yes, true

Reply:
Very happy to take a suggested reword. It also bears thinking about whether the left truncation adjust in particular is safe given it likely isn't used by most approaches that produce these estimates.